apps/scribe/src/configuration/configure-resolver.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  AlgoExplorerAdapter,
+  AlgorandAdapter,
+  CircleAdapter,
+  CMSCacheAdapter,
+  CoinbaseAdapter,
+  DirectusAdapter,
+  I18nAdapter,
+  MailerAdapter,
+  NFTStorageAdapter,
+} from '@algomart/shared/adapters'
+import {
+  AccountsService,
+  ApplicationService,
+  AuctionsService,
+  BidsService,
+  CollectiblesService,
+  CollectionsService,
+  HomepageService,
+  NotificationsService,
+  PacksService,
+  PaymentsService,
+  SetsService,
+  TransactionsService,
+} from '@algomart/shared/services'
+import { DependencyResolver } from '@algomart/shared/utils'
+import { Configuration } from './app-config'
+import { logger } from './logger'
+
+export function configureResolver() {
+  const resolver = new DependencyResolver()
+  resolver.set(
+    AlgorandAdapter.name,
+    () =>
+      new AlgorandAdapter(
+        {
+          algodPort: Configuration.algodPort,
+          algodServer: Configuration.algodServer,
+          algodToken: Configuration.algodToken,
+          fundingMnemonic: Configuration.fundingMnemonic,
+          appSecret: Configuration.secret,
+        },
+        logger
+      )
+  )
+  resolver.set(
+    AlgoExplorerAdapter.name,
+    () => new AlgoExplorerAdapter(Configuration.algodEnv, logger)
+  )
+  resolver.set(
+    CMSCacheAdapter.name,
+    () =>
+      new CMSCacheAdapter(
+        {
+          cmsUrl: Configuration.cmsUrl,
+          gcpCdnUrl: Configuration.gcpCdnUrl,
+        },
+        logger
+      )
+  )
+  resolver.set(
+    DirectusAdapter.name,
+    () =>
+      new DirectusAdapter(
+        {
+          cmsUrl: Configuration.cmsUrl,
+          gcpCdnUrl: Configuration.gcpCdnUrl,
+          accessToken: Configuration.cmsAccessToken,
+        },
+        logger
+      )
+  )
+  resolver.set(
+    NFTStorageAdapter.name,
+    () =>
+      new NFTStorageAdapter(
+        {
+          pinataApiKey: Configuration.pinataApiKey,
+          pinataApiSecret: Configuration.pinataApiSecret,
+          cmsUrl: Configuration.cmsUrl,
+          cmsPublicUrl: Configuration.cmsPublicUrl,
+          webUrl: Configuration.webUrl,
+        },
+        logger
+      )
+  )
+  resolver.set(
+    MailerAdapter.name,
+    () => new MailerAdapter(Configuration.mailer, logger)
+  )
+  resolver.set(
+    AccountsService.name,
+    (c) =>
+      new AccountsService(c.get<AlgorandAdapter>(AlgorandAdapter.name), logger)
+  )
+  resolver.set(
+    BidsService.name,
+    (c) =>
+      new BidsService(
+        c.get<NotificationsService>(NotificationsService.name),
+        c.get<PacksService>(PacksService.name),
+        Configuration.currency,
+        logger
+      )
+  )
```

Comment:
Currently seeing an error about the amount of arguments when I try to start up the app:

![Screen Shot 2022-04-05 at 10 57 47 AM](https://user-images.githubusercontent.com/13154207/161783444-d91ed93a-dbae-4ad8-96ae-73807e333c18.png)